bedrock/firefox/templates/firefox/australis/tour-base.html

```diff
@@ -23,8 +23,6 @@
 
 {% block body_class %}sky{% endblock %}
 
-{% block site_css %}{% endblock %}
```

Comment:
Pages using this base template (e.g. [firefox/australis/firstrun-tour.html](https://github.com/malena/bedrock/blob/bug-1171928-use-tabzilla-static-across-mozorg/bedrock/firefox/templates/firefox/australis/firstrun-tour.html#L27)) end up with flip-flopped CSS order. The page specific stuff ends up in `site_css`, while the common global stuff ends up in `page_css` (due to inheriting from [firefox/base.html](https://github.com/malena/bedrock/blob/bug-1171928-use-tabzilla-static-across-mozorg/bedrock/firefox/templates/firefox/base.html#L18)).

It's kind of a goose chase to follow the inheritance chain. :confused: However, you can rather easily hack the site to load a specific firstrun template by adding a line like `template = 'firefox/australis/firstrun-tour.html'` to [line 493 in `firefox/views.py`](https://github.com/malena/bedrock/blob/bug-1171928-use-tabzilla-static-across-mozorg/bedrock/firefox/views.py#L493).